Content.Server/Access/Systems/IdCardConsoleSystem.cs

```diff
@@ -197,10 +205,10 @@ private void UpdateStationRecord(EntityUid uid, EntityUid targetId, string newFu
         record.Name = newFullName;
         record.JobTitle = newJobTitle;
 
-        if (_prototype.TryIndex<JobPrototype>(newJobProto, out var job))
+        if (newJobProto != null)
         {
-            record.JobPrototype = newJobProto;
-            record.JobIcon = job.Icon;
+            record.JobPrototype = newJobProto.ID;
+            record.JobIcon = newJobProto.Icon;
         }
```

Comment:
Apparently record.JobIcon seems to be an RSI state, which it really shouldn't be. It should either be some prototypeId or a sprite specifier but I don't expect that to be changed in this PR, so I guess for now you need to try resolve the RSI state here.

Not directly related to this PR, but station records are looking pretty dubious in general. None of the fields are data-fields, so I don't even know if any of it gets saved?

Reply:
Nah I just rewrote everything to work with the proto id. Updating all the jobs was easier than undoing that.